### ISSUE TITLE
Speculative: Collect minutes of the Design Review Committee

### ISSUE THREAD

**Issue by bdougsand** [cityofsomerville/cornerwise#210]
The Design Review Committee meets once a month to discuss certain proposals. Can we get meaningful stuff from the documents they publish?

[Example](http://www.somervillema.gov/government/public-minutes)

**Comment by Bachmann1234** [comment]
Since this is marked speculative im poking around outside the cornerwise platform itself just to poke at the docs.

If it helps anyone I have downloaded the minutes for the past year (searching between july 28 2016 and july 28th 2015 for "Design Review Committee")

https://www.dropbox.com/sh/0zn1kfzw58zl8bo/AADqsA2e8HV2jZBw0s26M3Gga?dl=0

Im gonna poke at these and see what I learn dumping what I figure out into this directory.

**Comment by Bachmann1234** [comment]
Just updated by adding extracted versions of the pdf (script I used also in that dir)

**Comment by Bachmann1234** [comment]
So looking at the text it seems super doable to extract the address and the description of what happened. Im experienced with python and a lot of the tech you use in the project. But I am fairly ignorant to local government and the overall design of the project.

So I guess my questions are:
Is address and the description in these docs valuable to the project?
Can you give me a high level idea of your thoughts how this would fit in?

**Comment by danjmoore** [comment]
Thanks for looking into that, @Bachmann1234 !

I asked Dan Bartman (city planner) what the relationship between DRC topics and planning/zoning ones are. Essentially, most DRC items start off as planning/zoning requests and then go on to the DRC, but sometimes they start in the DRC. Either way, I think the info could be valuable to list out with others plans & docs. In cases where these topics start with the DRC, and hence have no existing entries on Cornerwise, maybe we could display the docs still and indicate that there hasn't been a permit request, but that it's something being considered at that location?

**Comment by Bachmann1234** [comment]
Here is a script I wrote to try and address info and the output after running on extracted pdfs from the past year
https://gist.github.com/Bachmann1234/3731ab2d7af6994833225687e0205059

Is this something that could be valuable? My only concern is that parsing of the PDF will be fragile. I tried not to get too specific in my parsing.

Obviously before attempting a PR I would add some tests and perhaps make a api of some kind to attempt to validate the address. Not to mention writing the job to check and download new meeting notes

Thoughts?
